internal/plugin/plugin.go

```diff
@@ -132,7 +134,10 @@ func (p *Plugin) GetCobraCommands() []*cobra.Command {
 		command := &cobra.Command{
 			Use:   command.Name,
 			Short: command.Description,
-			RunE:  p.Run,
+			Annotations: map[string]string{
+				"sourceType": SourceType,
+			},
```

Comment:
we should not reset the Annotations (can be used for help for example)
suggestion: Annotations: setSourceType(command.Annotations)
and setSourceType:
if cmd.Annotations == nil {
		cmd.Annotations = map[string]string{}
	}
	cmd.Annotations["test"] = "test"

Reply:
In this case we would not be reseting the annotations because the Command is just being created which means that there cannot be any annotations before that. Or am I missing something?

Reply:
I was thinking about carrying over the annotations from pluginCmd, but that a simple name/description struct, not a cobra command, so you can ignore my suggestion